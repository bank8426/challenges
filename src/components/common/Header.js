import React from 'react'
import styled from 'styled-components';
const StyledHeader = styled.h1`

`;
const Header = () => {
  return (
    <StyledHeader>
      Tamboon React
    </StyledHeader>
  )
}

export default Header
