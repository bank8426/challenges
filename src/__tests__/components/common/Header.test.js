import React from 'react'
import Header from '../../../components/common/Header'
import renderer from 'react-test-renderer'
import 'jest-styled-components'

it('should display header', () => {
  const tree = renderer.create(<Header />).toJSON()
  expect(tree).toMatchSnapshot()
})
