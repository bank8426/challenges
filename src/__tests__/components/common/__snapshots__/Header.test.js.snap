// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should display header 1`] = `
.c0 {
  background-color: #1B95F2;
  margin: 0;
  padding: 10px;
  color: white;
  text-align: center;
}

<h1
  className="c0"
>
  Tamboon React
</h1>
`;
